### PR TITLE
YML update

### PR DESCRIPTION
**Pull request by AkitaOnTon**
Please make sure you change the original ones `.yaml` fiels in directories `accounts/`, `collections/` or  `jettons/` and do not touch automatically generated .json files in the repository root.

Пожалуйста, убедитесь, что вы изменяете исходные `.yaml` файлы в директориях  `accounts/`, `collections/` или  `jettons/` и не трогаете автоматически сгенерированные файлы `.json` в корне репозитория.

### CODE REVIEW

**Comment by AkitaOnTon** [comment]
Hello this project is community takeover

**Comment by AkitaOnTon** [comment]
hello

**Comment by AkitaOnTon** [comment]
@oleganza @brainpicture @behrang @vladzima

**Comment by AkitaOnTon** [comment]
Our coin just created a bot for wallet tracker 
@AkitaTrackerBot (telegram)
And we just get coingecko 
https://www.coingecko.com/en/coins/akita-inu-3

File: jettons/AkitaOnTon.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: AkitaOnTon
+description: "The cutest dog on TON"
+image: "https://github.com/AkitaOnTon/AkitaOnTon.yml/blob/main/Akita.png"
```
Comment (aleksej-paschenko) [inline jettons/AkitaOnTon.yaml]: could you please update the link so it points to raw.githubusercontent.com?

**Review by aleksej-paschenko — CHANGES_REQUESTED** [review]

**Comment by AkitaOnTon** [comment]
@aleksej-paschenko I updated

**Comment by AkitaOnTon** [comment]
I needed to update one more thing, now everything is good.
@aleksej-paschenko

File: jettons/AkitaOnTon.yaml (hunk 1 above)
Comment (AkitaOnTon) [reply jettons/AkitaOnTon.yaml]: Ready to go

**Comment by aleksej-paschenko** [comment]
Thank you for the contribution, 
I changed its symbol to $AKITA because that is the exact symbol in the jetton's metadata and merged in another pr

https://github.com/tonkeeper/ton-assets/pull/242

**Pull request closed without merging**